core/app/models/spree/order/payments.rb

```diff
@@ -31,10 +31,6 @@ def capture_payments!
           process_payments_with(:purchase!)
         end
 
-        def pending_payments
```

Comment:
I would prefer that this method remain, and be marked as deprecated and that is is scheduled for removal in a future version of solidus, telling the developer to use `payments.pending` instead.